src/Pim/Bundle/CatalogBundle/Elasticsearch/Sorter/Field/FamilySorter.php

```diff
@@ -53,8 +53,9 @@ public function addFieldSorter($field, $direction, $locale = null, $channel = nu
                 if (null !== $familyLabelPath) {
                     $sortFamilyLabelClause = [
                         $familyLabelPath => [
-                            'order'   => 'ASC',
-                            'missing' => '_last',
+                            'order'         => 'ASC',
```

Comment:
You will need to fix specs FamilySorterSpec and FamilySorterIntegration tests also